src/citrine/resources/project.py

```diff
@@ -249,6 +353,9 @@ def publish(self, *, resource: Resource):
         """
         resource_access = resource.access_control_dict()
         resource_type = resource_access["type"]
+        if resource_type == ResourceTypeEnum.DATASET:
+            warn("Datasets are no longer owned by Projects, so cannot be published by a Project.",
```

Comment:
Shouldn't this warning explicitly call out the Team-based route?

Reply:
"Publishing" no longer means anything with regards to datasets in the world of Data Manager. Previously, it's what made them visible to the team so another project could pull them in. Now, since everything is at a team level, it's a no-op.

Sounds like the message should probably be changed to be a bit clearer.

Reply:
Yes, publishing doesn't mean anything now for team-based datasets, however, the accounts service doesn't implement an NOP for this type of dataset. Instead, it returns a 403, because the project doesn't have admin rights on the new type of dataset. I'd say that on top of the warning we should not call the API at all.